testing_utils_test.go

```diff
@@ -79,22 +78,28 @@ func getTestURL(urlString string) string {
 	return fmt.Sprintf("%s://%s", parsedURL.Scheme, strings.Join([]string{parsedURL.Host, parsedURL.Host, parsedURL.Host}, ","))
 }
 
+func newFakeAuthMarathonEndpoint(t *testing.T, config *Config, username, password string) *endpoint {
```

Comment:
With the very recent merge of #158, I think we should take advantage of the new `ServerConfig` type and amend it by `username` and `password` fields. That way, we can reuse the existing method calls and don't have to define `newFakeAuthMarathonEndpoint`.